src/app/lib/layoutGrid.js

```diff
@@ -40,7 +64,8 @@ export const gelGridMargin = css`
 */
 
 export const layoutGridWrapper = css`
-  display: grid;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Replacing these two lines with one:
```suggestion
  margin: 0 auto;
```